code/game/machinery/computer/arcade.dm

```diff
@@ -703,6 +763,7 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 				fuel -= 10
 				food -= 10
 				event()
+				killed_crew-- // I mean not really but you know
```

Comment:
what if someone is buying crew members just to kill them

Reply:
As long as they kill everyone they meet, they'll still hit the required counter. This ensures that everyone had to die by getting BLAM'd, rather than 4 BLAMs and some other cause of death cause you bought an extra dude